app/views/unattended/provisioning_templates/provision/atomic_kickstart_default.erb

```diff
@@ -42,16 +42,51 @@ rootpw --iscrypted <%= root_pass %>
 
 reboot
 
+<% if @dynamic -%>
+# Partition table dynamic generation
+%pre --log=/tmp/install.pre.dynamic.log
+<%= @host.diskLayout %>
+<%= section_end -%>
```

Comment:
You need to set this variable. It's not in this file right now, unlike the default kickstart.

Reply:
Nice find.